vmdb/db/migrate/20150507102612_create_container_node_conditions.rb

```diff
@@ -0,0 +1,17 @@
+class CreateContainerNodeConditions < ActiveRecord::Migration
+  def up
+    create_table :container_node_conditions do |t|
+      t.belongs_to :container_node, :type => :bigint
```

Comment:
@abonas I don't believe this should be an ems entity, these are more similar to labels and the name is enough (given a `container_node`).